docs/markdown/Machine-files.md

```diff
@@ -84,14 +84,63 @@ command line will override any options in the native file. For example, passing
 
 In addition to special data that may be specified in cross files, this
 section may contain random key value pairs accessed using the
-`meson.get_external_property()`
+`meson.get_external_property`, or `meson.get_cross_property`.
 
-## Properties
+### Project specific options
 
-*New for native files in 0.54.0*
+*New in 0.55.0*
 
-The properties section can contain any variable you like, and is accessed via
-`meson.get_external_property`, or `meson.get_cross_property`.
+Path options are not allowed, those must be set in the `[paths]` section.
+
+Being able to set project specific options in a cross or native file can be
+done using the `[project options]` section of the specific file (if doing a
+cross build the options from the native file will be ignored)
+
+An incomplete list of options is:
+- pkg_config_path
+- cmake_prefix_path
+
+For setting options in supbprojects use the `<subproject>:project options`
+section instead.
+
+```ini
+[project options]
+build-tests = true
```

Comment:
IIRC the cross file parser force to have quotes for all values, do you have a unit test that verify that actually works?